### ISSUE TITLE
[Feature Request] Option to open catalog instead of the first page when opening a board 

### ISSUE THREAD

**Issue by PwnicornDev** [miku-nyan/Overchan-Android#93]
I always use the catalog and the extra step of choosing catalog in the menu after opening the board is annoying.
Thank you for this great app and the continuous support, it's very much appreciated.

**Comment by James-E-A** [comment]
Agreed, with the thread previews not available at a glance, the "Pages" are just a straight "downgrade" from the catalog.
